snap7/server.py

```diff
@@ -242,7 +244,8 @@ def set_mask(self, kind: int, mask: int):
     def set_cpu_status(self, status: int):
         """Sets the Virtual CPU status.
         """
-        assert status in snap7.types.cpu_statuses, f'unknown cpu state {status}'
+        if status not in snap7.types.cpu_statuses:
+            raise Snap7Exception(f"The cpu state ({status}) is invalid")
```

Comment:
sorry, i think we should probably just get rid of this snap7exection. I think this is a value error.